core/lib/Thelia/Config/I18n/en_US.php

```diff
@@ -55,7 +55,7 @@
     'Associate documents' => 'Associate documents',
     'Associate images' => 'Associate images',
     'At least one of cart products categories is %op% <strong>%categories_list%</strong>' => 'At least one of cart products categories is %op% <strong>%categories_list%</strong>',
-    'Attribut' => 'Attribut',
+    'Attribut' => 'Attribute',
```

Comment:
Why ?? 
The replacement for other languages is not so difficult. And this key could be used as a fallback if a translation is not found.